### PR TITLE
Fix some typos/small mistakes

### PR DESCRIPTION
**Pull request by zlangley**

### CODE REVIEW

File: docs/build-apps/apps.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ To build a payment-based application, a firm understanding of how to construct a
...
 
-Algorand Standard Assets are the underlying technology that support these applications on Algorand and are deployed at Layer-1 through a simple transaction (no contract code is required). Assets can be traded like the Algo -  Algorand’s native token. They are also highly configurable, providing capabilities for minting, freezing, and revoking tokens.  For more information on building and deploying asset-based applications see the [Algorand Standard Assets](../features/asa.md) feature guide.
```
Comment (algorandskiy) [inline docs/build-apps/apps.md:22]: this fix is incorrect - a dash is needed here

Comment (zlangley) [reply docs/build-apps/apps.md:22]: I replaced the hyphen ("-") with an em-dash ("—"), though it's hard to tell in this monospaced font. That is, I changed "the Algo-Algorand's native token" to "the Algo—Algorand's native token". The former looks like a hyphenated noun (an "Algo-Algorand") whereas the latter is clearly a break in the sentence.

Comment (algorandskiy) [reply docs/build-apps/apps.md:22]: ok, but maybe keep spaces around?

Comment (zlangley) [reply docs/build-apps/apps.md:22]: Okay, done. (Though I prefer it without the spaces. 🙃)

**Review by algorandskiy — APPROVED** [review]
Thanks, now it looks like em-dash

**Pull request merged**